templates/admin/badge/users.tmpl

```diff
@@ -0,0 +1,41 @@
+{{template "admin/layout_head" (dict "ctxData" . "pageClass" "admin badge")}}
+	<div class="admin-setting-content">
+		<h4 class="ui top attached header">
+			{{.Title}}
+		</h4>
+		{{if .Users}}
+		<div class="ui attached segment">
+			<div class="flex-list">
+				{{range .Users}}
+					<div class="flex-item tw-items-center">
+						<div class="flex-item-leading">
+							<a href="{{.HomeLink}}">{{ctx.AvatarUtils.Avatar . 32}}</a>
+						</div>
+						<div class="flex-item-main">
+							<div class="flex-item-title">
+								{{template "shared/user/name" .}}
+							</div>
+						</div>
+						<div class="flex-item-trailing">
+							<a class="ui red tiny button inline link-action" data-url="{{$.Link}}/delete?id={{.ID}}" data-modal-confirm="{{ctx.Locale.Tr "admin.badges.delete_user_desc"}}">
+								{{ctx.Locale.Tr "admin.badges.remove_user"}}
+							</a>
+						</div>
+					</div>
+				{{end}}
+			</div>
+		</div>
+		{{end}}
+		{{template "base/paginate" .}}
+		<div class="ui bottom attached segment">
+			<form class="ui form" id="search-badge-user-form" action="{{.Link}}" method="post">
```

Comment:
Why the search form is at the bottom?

IIRC for all (at least almost) cases Gitea layout is like this: search form at top, table at bottom.


ps: why it needs an `id`? I don't see it is used anywhere.

Reply:
Hmm ... it is not a real "search" form, but it is used to add a new user, so maybe it's fine.

But still have the question why there are unused `id` attributes.